app/messaging/outbound/register-import-request.js

```diff
@@ -0,0 +1,18 @@
+const { v4: uuidv4 } = require('uuid')
+
+const createMessage = (data) => ({
+  body: {
+    specVersion: '1.0',
+    type: 'REIGSTER_IMPORT',
+    source: 'aphw-ddi-portal',
+    id: uuidv4(),
+    time: new Date().toISOString(),
```

Comment:
Should we be using UTC?

Reply:
Yes that's right. ISO uses UTC but there can be a optional time zone offset which we don't need.